tests/Controller/StatusControllerTest.php

```diff
@@ -2,6 +2,9 @@
 
 namespace Tests\BBC\CliftonBundle\Controller;
 
+use BBC\ProgrammesPagesService\Domain\ValueObject\Pid;
+use Doctrine\DBAL\ConnectionException;
+use Doctrine\DBAL\DBALException;
```

Comment:
I dislike the group-import thing as it makes searching for usage of a class a bit more fiddly. One imported thing per line imo :)